ts-sdk/whirlpool/src/sysvar.ts

```diff
@@ -12,17 +10,15 @@ const ACCOUNT_STORAGE_OVERHEAD = 128;
  *
  * @param {Rpc} rpc - The Solana RPC client to fetch sysvar rent data.
  * @param {number} dataSize - The size of the account data in bytes.
- * @returns {Promise<BigInt>} The minimum balance required for rent exemption in lamports.
+ * @returns {bigint} The minimum balance required for rent exemption in lamports.
  */
-export async function calculateMinimumBalance(
-  rpc: Rpc<GetAccountInfoApi>,
+export function calculateMinimumBalance(
```

Comment:
nit: maybe we can be a bit more descriptive with this naming. `getMinimumBalanceForRentExemption` is what the rpc method that does the same is called